pages/index.js

```diff
@@ -103,6 +113,18 @@ export default function HomePage({ onToggleFavourites, favouriteIdeas }) {
               </CardListItem>
             ))}
       </CardList>
+      <ButtonBox>
+        <PreviousButton
+          onClick={() => setCurrentPage(currentPage - 1)}
+          show={currentPage > 1}
+        >
+          Previous Page
+        </PreviousButton>
+
+        <button onClick={() => setCurrentPage(currentPage + 1)}>
+          Next Page
+        </button>
```

Comment:
The "Next" button is always displayed. The user can click it endlessly and gets  blank pages/views 😉